mine/macros/generic_tests/null_proportion.sql

```diff
@@ -0,0 +1,21 @@
+{% macro test_null_proportion(model) %}
```

Comment:
It's simplest to pass arguments to your macro here: e.g. model, column_name, at_most. Try that structure instead of the more complex kwargs.get!